seatunnel-connectors-v2/connector-jdbc/pom.xml

```diff
@@ -29,13 +29,6 @@
 
     <artifactId>connector-jdbc</artifactId>
 
-    <properties>
-        <phoenix.version>5.2.5-HBase-2.x</phoenix.version>
-        <mysql.version>8.0.16</mysql.version>
-        <postgresql.version>42.3.3</postgresql.version>
-        <dm-jdbc.version>8.1.2.141</dm-jdbc.version>
-    </properties>
-
```

Comment:
Don‘t manage the properties of the connector in root pom.xml

That's my suggestion:

```xml
    <properties>
        <phoenix.version>5.2.5-HBase-2.x</phoenix.version>
        <mysql.version>8.0.16</mysql.version>
        <postgresql.version>42.3.3</postgresql.version>
        <dm-jdbc.version>8.1.2.141</dm-jdbc.version>
    </properties>

    <dependencies>
        <dependency>
            <groupId>com.aliyun.phoenix</groupId>
            <artifactId>ali-phoenix-shaded-thin-client</artifactId>
        </dependency>
    </dependencies>

    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>com.aliyun.phoenix</groupId>
                <artifactId>ali-phoenix-shaded-thin-client</artifactId>
                <version>${phoenix.version}</version>
            </dependency>
        </dependencies>
    </dependencyManagement>
```

This is then used in the E2E module

```xml
    <dependencies>
        <dependency>
            <groupId>com.aliyun.phoenix</groupId>
            <artifactId>ali-phoenix-shaded-thin-client</artifactId>
            <scope>test</scope>
        </dependency>
    </dependencies>

    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>org.apache.seatunnel</groupId>
                <artifactId>connector-jdbc</artifactId>
                <version>${project.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
```